### PR TITLE
colw/Show loading and not found notices for a single validator

### PR DESCRIPTION
**Pull request by colw**
colw/Show loading and not found notices for a single validator

Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -1,12 +1,18 @@
 <template>
   <TmPage
     :managed="true"
-    :data-empty="!validator.operatorAddress"
     :hide-header="true"
     data-title="Validator"
     class="small"
   >
-    <template v-if="validator.operatorAddress" slot="managed-body">
+    <template
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue]: Please use the TmPage logic so this is standardized. Alternatively refactor the logic out of TmPage and create a DataLoading component.
I think we shouldn't duplicate the logic.

File: src/components/staking/PageValidator.vue [hunk 2]
```diff
@@ -266,7 +293,8 @@ export default {
         }
       `,
       skip() {
-        return !this.userAddress
+        !this.userAddress ||
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue]: Could be refactored into a computed

**Comment by colw** [comment]
I reverted #8d1add3ca38f9d8692a3c9397e2323cc3916d58f as it broke unrelated things. What was wrong with that? It should return true for the testnet?

File: src/ActionModal/components/ActionModal.vue [hunk 3]
```diff
@@ -415,6 +415,11 @@ export default {
     ...mapGetters([`connected`, `isExtensionAccount`]),
     ...mapGetters({ networkId: `network` }),
     checkFeatureAvailable() {
+      /* istanbul ignore next */
+      if (this.network.testnet) {
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue]: no no no no ^^

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]:
```suggestion
      if (this.network.id === 'local-cosmos-hub-testnet') {
```

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: `gaia-13006` is also a `testnet`

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: It would be the best option if this would be solved via the API I think

Comment (colw) [reply src/ActionModal/components/ActionModal.vue]: Good point. I remember we discussed this the other day too. 👍

**Comment by colw** [comment]
Closed in favour of #3123

**Pull request closed without merging**